setup.cfg

```diff
@@ -2,10 +2,87 @@
 #
 # This file is part of Invenio.
 # Copyright (C) 2016-2019 CERN.
+# Copyright (C) 2025 Northwestern University.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+[metadata]
+name = invenio-records-files
+version = attr: invenio_records_files.__version__
+description = Integrates Invenio records with files
+long_description = file: README.rst, CHANGES.rst
+keywords = invenio records files
+license = MIT
+author = CERN
+author_email = info@inveniosoftware.org
+platforms = any
+url = https://github.com/inveniosoftware/invenio-records-files
+classifiers =
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Topic :: Internet :: WWW/HTTP :: Dynamic Content
+    Topic :: Software Development :: Libraries :: Python Modules
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
+    Programming Language :: Python :: 3.13
+    Programming Language :: Python :: 3.14
+    Programming Language :: Python :: Implementation :: CPython
+    Development Status :: 5 - Production/Stable
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.9
```

Comment:
It used to be 3.6 according to the classifiers... Even 3.9 is deprecated at this point, but this is a compromise because big installations are still on 3.9 (freshly deprecated). At least moves to parity with other modules.

As such, I am undecided about what kind of version bump is warranted for this package  :

- Not breaking functionality interface, but might be breaking installation interface
- Not adding new functionality, but are adding new installation functionality/different version branch for updates
- Just patching supported Python versions, but that can be a big change as above

Maybe minor is the compromise?